Update VM-19 per PM Vikas Mohandoss

pinyuko opened the pull request:
<!-- Thank you for submitting a Pull Request. Please fill out the template below.-->
# Overview/Summary
Checked with Disk PM for Encryption Vikas Mohandoss, updating this per his feedback.

## Related Issues/Work Items

Replace this with a list of related GitHub Issues and/or ADO Work Items (Internal Only)

- To associate a GitHub Issue, use a [key word](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) preceded with the GitHub issue number.
- To associate an ADO Work Item, use the key word `AB#` succeeded with the [ADO Work Item ID](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## This PR fixes/adds/changes/removes

1. Remove the existing query since it's incorrect and list as under development for now. Vikas will get back later
2. Update the entire VM-19 as below:
Title: Enable advanced encryption options for your managed disks
Azure Disk Storage Server-Side Encryption (also referred to as encryption-at-rest or Azure Storage encryption) automatically encrypts data stored on Azure managed disks (OS and data disks) when persisting on the Storage Clusters. There are several types of advanced encryption options available for your managed disks, including Azure Disk Encryption (ADE), Encryption at host and Confidential disk encryption.
•	ADE encrypts the disks of Azure virtual machines (VMs) inside your VMs by using the DM-Crypt feature of Linux or the BitLocker feature of Windows.
•	Encryption at host ensures that data stored on the VM host hosting your VM is encrypted at rest and flows encrypted to the Storage clusters.
•	Confidential disk encryption binds disk encryption keys to the virtual machine’s TPM and makes the protected disk content accessible only to the VM.

### Breaking Changes

1. As above 1. Remove VM-19 query.

## As part of this Pull Request I have

- [X] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [X] Ensured PR tests are passing
- [X] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

ejhenry commented on docs/content/services/compute/virtual-machines/code/vm-19/vm-19.kql:
Hi @pinyuko, rather than changing the content of this file, can you change the file extension to .fix and create a new vm-19.kql with the // under-development text? That way the original query is available for anybody that would like to fix it.

pinyuko commented:
Changed the scripts that needs to be fixed to be "VM-19.fix", and changed the VM-19.kql to be // under-development since PM mentioned the query does not align with the updated recommendation.

Ready to be merged by APRL team.

edknox reviewed (approved):
With .fix file it looks good
